### DISCUSSION TITLE
v2.8.2 on Mac with Hottop K+, Slider numeric display no longer shows 5 percent Air and Burner power changes

### DISCUSSION THREAD

**Discussion by Javapik** [artisan-roaster-scope/artisan#1157] [Q&A]
Greetings,
Installed 2.8.2 and lost Air & Burner  +5 &-5 increments on the numeric display above the sliders. It looks like the slider can move by 5's but the LCD only displays changes in increments of 10. This happens when I used the mapped buttons or grab the sliders and move them manually.  I tried rebuilding the buttons but that didn't help. Is there some other setting or action I need to take? For now I've gone back to using Artisan vs 2.6.0 on my MacBook Pro (13", Early 2015) running OS Big Sur (11.6.4)
Thanks in advance, Ed.

**Comment by MAKOMO** [comment]
The Air slider moves in steps of 10 (a limit of the Hottops hardware, not of Artisan) and the Burner slider in steps of 1. Sliders and LCDs updates are always synchronized.

There is no difference in Hottop support between v2.6.0 and v2.8.2.

**Reply by Javapik** [reply]
Yes, my mistake in typing on the Air, it's the Burner when using the +5 or -5 button that the numeric readout doesn't follow. I'll make a video tomorrow if you like.

**Reply by Javapik** [reply]
Hmmm, having trouble uploading videos from my iPhone. Is there a link with instructions? Tried 2.8.2 again and still can’t get the -5 button to show increments of 5 on the numeric display.

**Reply by Javapik** [reply]
Posted it to YouTube so you could see it: https://youtu.be/vZm7CBVyiWM

**Comment by Javapik** [comment]
Do you need me to post the video of version 2.6 correctly displaying changes of 5?

**Reply by poundy** [reply]
you should, yes, so we know there's actually a repro where it was working. 

As for phone and video, using YT is a perfectly acceptable way.

**Comment by MAKOMO** [comment]
You are not using the default Hottop button setup. Please export your Artisan settings via menu Help >> Export settings into a file artisan-settings.aset. Rename this file into artisan-settings.txt and then drag and drop this file to a comment box here on this discussion on GitHub. I will take a look at your button definition and check what is wrong. Thanks!

**Reply by Javapik** [reply]
I will do that in a couple of days.

**Reply by Javapik** [reply]
Here is  version  2.6.0 that like previous versions works with  changes of  5 https://youtu.be/Zw_isjZFTUA

**Comment by Javapik** [comment]
Here's a YouTube of  vs 2.6.0 that like previous versions correctly correctly changes of  5 with the  Burner slide : https://youtu.be/Zw_isjZFTUA

**Comment by MAKOMO** [comment]
Could you please also attach your artisan-settings.aset file (renamed with .txt extension) here? Thanks!

**Reply by MAKOMO** [reply]
alternatively you could put here a screenshot of your buttons definitions (menu Config >> Events, tab Buttons)

**Reply by Javapik** [reply]
Hundreds of miles away from it. Will send it upon my return.

**Reply by Javapik** [reply]
Marko, just returned. I opened the Events Tab and saw that the Burner -5 Button label was different in 2.8.2 for some unknown reason. It read "\+\n-5" instead of "Burner\n-5". I've now changed it to read the same as the 2.6.0 so I'll try it tomorrow and see if that fixes it.
![Screen Shot 2023-05-11 at 3 14 15 PM](https://github.com/artisan-roaster-scope/artisan/assets/94915335/478d8630-152a-4d99-be1d-004cfd5a6f34)

**Reply by Javapik** [reply]
Darn, there was a typo: It read "\t\n-5" not "\+\n-5"

**Reply by MAKOMO** [reply]
This is a confusing report. Button labels do not influence button functionality. However, what I read between the lines is that the button setups between the two tested Artisan versions were different and thus the behavior. You did not identify a bug here that needs to be fixed, right?

**Reply by Javapik** [reply]
Powered up the roaster this morning and verified  that changing the Button label did NOT fix the problem. By the way, the Controls LCD for the Heater on the right side of the screen, which was not in frame of the video, DOES display the increments of 5.
Thanks, Ed

PS: Is it normal that I always have to restart the computer for it to link to the  Hottop?
